src/parser.rs

```diff
@@ -17,11 +17,13 @@ pub fn parse_text(s: &str) -> Result<Expr, Error> {
     let mut pairs = CQL2Parser::parse(Rule::Expr, s).map_err(Box::new)?;
     if let Some(pair) = pairs.next() {
         if pairs.next().is_some() {
+            dbg!(1, s, pairs);
             Err(Error::InvalidCql2Text(s.to_string()))
         } else {
             parse_expr(pair.into_inner())
         }
     } else {
+        dbg!(2, s, pairs);
```

Comment:
If we want logging, we should use [log](https://docs.rs/log/latest/log/):

```suggestion
```

Reply:
oh shoot, meant to get rid of those statements, they were just there while I was debugging, I'll remove